MUXSDKKaltura/MUXSDKKaltura/MUXSDKStats.swift

```diff
@@ -125,6 +125,22 @@ public class MUXSDKStats: NSObject {
     public static func destroyPlayer(name: String) {
         self.bindingsManager.destroyPlayer(name: name)
     }
+    
+    /**
+     Dispatches an error with the specified error code and message for the given player
+     
+     - Parameters:
+        - name: The name of the player
+        - code The error code in string format
+        - message: The error message in string format
+     */
+    public static func dispatchErrorForPlayer(name: String, code: String, message: String) {
+        guard let binding = bindingsManager.bindings[name] else {
+            return
```

Comment:
maybe we should log something here?

Reply:
we're not logging anything on avPlayer but we can change that if we thing it would be useful

Reply:
logging seems like a good idea here